src/main/java/UI/CheckoutPage.java

```diff
@@ -11,84 +11,89 @@
 import java.time.LocalDate;
 
 public class CheckoutPage extends Page implements ActionListener {
-    final int WIDTH = 800;
-    final int HEIGHT = 600;
-    private final JButton SUBMIT;
-    JTextField jtUsername = new JTextField();
-    JTextField jtName = new JTextField();
-    JTextField jtCardNumber = new JTextField(16);
-    JTextField jtCVV = new JTextField(3);
-    JTextField jtExpiration = new JTextField();
-    JTextField jtAddress = new JTextField();
+    private final JButton SUBMIT = new JButton("Submit");
+    private JTextField jtUsername = new JTextField();
+    private JTextField jtName = new JTextField();
+    private JTextField jtCardNumber = new JTextField(16);
+    private JTextField jtCVV = new JTextField(3);
+    private JTextField jtExpiration = new JTextField();
+    private JTextField jtAddress = new JTextField();
+    private final SpringLayout LAYOUT;
+    private LabelTextPanel usernameInfo;
+    private LabelTextPanel nameInfo;
+    private LabelTextPanel cardNumberInfo;
+    private LabelTextPanel cvvInfo;
+    private LabelTextPanel expirationInfo;
+    private LabelTextPanel addressInfo;
+    private JLabel titleLabel;
 
     public CheckoutPage() {
-        super("Checkout");
+        super("Scamazon.ca");
+        this.LAYOUT = new SpringLayout();
 
-        JPanel form = new JPanel();
-        JFrame frame = new JFrame();
-        frame.setMinimumSize(new Dimension(WIDTH, HEIGHT));
-        frame.setTitle("Scamazon.ca");
-        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.add(form);
-
-        form.setLayout(new BorderLayout(10, 10));
-        form.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
-
-        JLabel username = new JLabel("Username");
-        username.setBounds(10, 20, 80, 25);
-        form.add(username);
-        jtUsername.setBounds(100, 20, 165, 25);
-        form.add(jtUsername);
+        setUpPanel();
```

Comment:
really like how u extracted these from the constructor, much more readable this way